### PR TITLE
Describe team roles alongside org member roles

### PR DESCRIPTION
**Pull request by RyanSkonnord**
Add description of team roles. Split table of organization membership in two, deleting the "admin" column and moving affected rows to a new "Team Member Roles" subsection. Demoted the "Organization Member Roles" section to an `h3` so that "Team Member Roles" can sit next to it.

### CODE REVIEW

File: src/docs/product/accounts/membership.mdx [hunk 1]
```diff
@@ -15,51 +15,66 @@ which can then join one or more organizations. Each user should have
...
+| Can add/remove Repositories  | X                | X          |
+
+#### Team Admin
+
+- Can create new projects and remove projects they're a member of.
+
+#### Team Contributor
+
+- Can view and act on events, as well as view most other data within the team's
+  projects.
```
Comment (RyanSkonnord) [inline src/docs/product/accounts/membership.mdx:76]: I wrote these for symmetry with the "Organization Member Roles" section, but they seem a little unnecessary and can maybe just be deleted?

Comment (armenzg) [reply src/docs/product/accounts/membership.mdx:76]: I would be fine with that.

**Review by armenzg — APPROVED** [review]

File: src/docs/product/accounts/membership.mdx (hunk 1 above)
Comment (RyanSkonnord) [reply src/docs/product/accounts/membership.mdx]: Done.

File: src/docs/product/accounts/membership.mdx [hunk 2]
```diff
@@ -15,51 +15,57 @@ which can then join one or more organizations. Each user should have
...
-## Roles
+### Organization Member Roles
 
 Member roles dictate access within an organization.
 
-### Owner
+| Action                                                       | Billing | Member | Manager | Owner |
+| ------------------------------------------------------------ | ------- | ------ | ------- | ----- |
+| Can see/edit billing information and subscription details    | X       |        |         | X     |
+| Can see/edit legal and compliance details                    | X       |        |         | X     |
```
Comment (imatwawana) [inline src/docs/product/accounts/membership.mdx:25]:
```suggestion
| Can view/edit billing information and subscription details    | X       |        |         | X     |
| Can view/edit legal and compliance details                    | X       |        |         | X     |
```

File: src/docs/product/accounts/membership.mdx [hunk 3]
```diff
@@ -15,51 +15,57 @@ which can then join one or more organizations. Each user should have
...
+| Can view and act on issues, such as assigning/resolving/etc. |         | X      | X       | X     |
+| Can join and leave teams                                     |         | X      | X       | X     |
+| Is automatically a Team Admin                                |         |        | X       | X     |
```
Comment (imatwawana) [inline src/docs/product/accounts/membership.mdx:28]:
```suggestion
| Is automatically a team admin                                |         |        | X       | X     |
```

File: src/docs/product/accounts/membership.mdx [hunk 4]
```diff
@@ -15,51 +15,57 @@ which can then join one or more organizations. Each user should have
...
+| Can add/remove teams                                         |         |        | X       | X     |
+| Can change Organization Settings                             |         |        | X       | X     |
+| Can remove an Organization                                   |         |        |         | X     |
```
Comment (imatwawana) [inline src/docs/product/accounts/membership.mdx:31]:
```suggestion
| Can change organization settings                             |         |        | X       | X     |
| Can remove an organization                                   |         |        |         | X     |
```

File: src/docs/product/accounts/membership.mdx [hunk 5]
```diff
@@ -15,51 +15,57 @@ which can then join one or more organizations. Each user should have
...
 
 - Unrestricted access to the organization, its data, and settings.
 - Can add, modify, and delete projects and members, as well as manage subscription and billing details.
 - Can manage legal and compliance details.
 - Can delete an organization.
 
-### Manager
+#### Manager
 
 - Gains admin access on all teams.
```
Comment (imatwawana) [inline src/docs/product/accounts/membership.mdx:42]:
```suggestion
- Has admin access on all teams.
```

File: src/docs/product/accounts/membership.mdx [hunk 6]
```diff
@@ -15,51 +15,57 @@ which can then join one or more organizations. Each user should have
...
 
 - Can manage subscription and billing details.
 - Can manage legal and compliance details.
 
+### Team Member Roles
+
+Team roles dictate access within projects owned by the team.
+
+A project may belong to multiple teams. To have administrative access to a
+project, you only need to be an admin of one of those teams, not all.
```
Comment (imatwawana) [inline src/docs/product/accounts/membership.mdx:60]:
```suggestion
project, you only need to be an admin of one of those teams, not all of them.
```

File: src/docs/product/accounts/membership.mdx [hunk 7]
```diff
@@ -15,51 +15,57 @@ which can then join one or more organizations. Each user should have
...
+
+| Action                       | Team Contributor | Team Admin |
+| ---------------------------- | ---------------- | ---------- |
+| Can change Project Settings  |                  | X          |
```
Comment (imatwawana) [inline src/docs/product/accounts/membership.mdx:64]:
```suggestion
| Can change project settings  |                  | X          |
```

File: src/docs/product/accounts/membership.mdx [hunk 8]
```diff
@@ -15,51 +15,57 @@ which can then join one or more organizations. Each user should have
...
+| Can add/remove projects      |                  | X          |
+| Can edit Global Integrations |                  | X          |
+| Can add/remove Repositories  | X                | X          |
```
Comment (imatwawana) [inline src/docs/product/accounts/membership.mdx:67]:
```suggestion
| Can edit global integrations |                  | X          |
| Can add/remove repositories  | X                | X          |
```

**Review by imatwawana — APPROVED** [review]
Looks good to me! I made some wording edits and removed some unnecessary capitalization, but otherwise LGTM.

**Pull request closed without merging**